pkg/pillar/zedbox/zedbox.go

```diff
@@ -39,12 +41,17 @@ func main() {
 	switch basename {
 	case "client":
 		client.Run(ps)
+	case "command":
+		command.Run()
 	case "diag":
 		diag.Run(ps)
 	case "domainmgr":
 		domainmgr.Run(ps)
 	case "downloader":
 		downloader.Run(ps)
+	case "executor":
+		// XXX add ps
+		executor.Run()
```

Comment:
Please pass it `PubSub.pubsub` to keep with what the others do.

Reply:
Done.

Reply:
Thanks